src/domain/occurrence/eventOccurrenceForm/EventOccurrenceForm.tsx

```diff
@@ -87,6 +89,31 @@ const EventOccurrenceForm: React.FC<Props & GoToPublishingProps> = ({
     Boolean(initialValues.placeId)
   );
 
+  const getLanguagesSorted = () => {
+    const languagesOrder = sortFavorably(
+      Object.values(EVENT_LANGUAGES).map((language) =>
+        t(`common.languages.${language.toLowerCase()}`)
+      ),
+      [
+        EVENT_LANGUAGES.FI,
+        EVENT_LANGUAGES.SV,
+        EVENT_LANGUAGES.EN,
+      ].map((language) => t(`common.languages.${language.toLowerCase()}`))
+    );
+
+    const languages = [
+      ...Object.values(EVENT_LANGUAGES).map((language) => ({
```

Comment:
Can you use languagesOrder here so that we wouldn't need another sort?

Reply:
The problem is that `languagesOrder` does not contain the value - it's a list of labels. I tried to create an alternative for `sortFavorably` that would have sorted objects with a given key, but that would have been a mess in TypeScript. Since it's needed only here, I thought it would be better to do it like this.

The thing with the complex sorter is that there are 6 languages total, but we want to add 3 of them (fi, sv, en) in the beginning of the array and rest should be sorted alphabetically (..., ar, ru, zh).

Reply:
Just thinking if that would just work correctly? Or is it wrong order, can't test it at the moment :D

```
languagesOrder.map((language) => ({
        label: t(`common.languages.${language.toLowerCase()}`),
        value: language,
      }))
```